ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/TlsIT.scala

```diff
@@ -42,8 +46,12 @@ final class TLSOnePointThreeIT
   * - accepts either TLSv1.2 or TLSv1.3 connections,
   * - rejects TLSv1.1 (or lower) connections.
   */
-final class TLSAtLeastOnePointTwoIT
-    extends TlsIT(shortIdentifierPrefix = "ServerOnTLSConnectionFromClientOn") {
+final class TLSAtLeastOnePointTwoIT(
+    clientTlsConfiguration: Option[TlsConfiguration]
```

Comment:
Nice change with passing the client TlsConfiguration to the test suite constructor.
At this point we already know that is should be present, so I'm wondering whether it makes sense to pass it as optional (since `None` is never valid in this context anyway)

Reply:
Oh, actually we are building the optional tests eagerly even though they might not be targeted by an inclusion, so no client tls configured..